doc/en/api/1.0.0/gwcmasstruncate.yaml

```diff
@@ -0,0 +1,48 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: GeoWebCache Mass Truncate
+  description: The REST API for mass truncation provides a mechanism for completely clearing caches more conveniently than with the seeding system
+schemes:
+- http
+host: localhost:8080
+basePath: /geoserver/gwc/rest
+paths:
+  /masstruncate:
+    get:
+      operationId: masstruncateGet
+      summary: Returns available request types for truncation
+      description: Returns xml containing the request type capabilities for mass truncation
+      produces:
+        - application/xml
```

Comment:
Most other endpoints support both json and xml from what I can see. Is this an ommission, or does this endpoint really only support XML

Reply:
This will accept a json extension in the request, but it will still respond/produce xml.
```curl -v -u admin:geoserver -X GET "http://localhost:8080/geoserver/gwc/rest/masstruncate.json"
Note: Unnecessary use of -X or --request, GET is already inferred.
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8080 (#0)
* Server auth using Basic with user 'admin'
> GET /geoserver/gwc/rest/masstruncate.json HTTP/1.1
> Host: localhost:8080
> Authorization: Basic YWRtaW46Z2Vvc2VydmVy
> User-Agent: curl/7.54.0
> Accept: */*
> 
< HTTP/1.1 200 OK
< Date: Mon, 25 Sep 2017 17:20:19 GMT
< X-Frame-Options: SAMEORIGIN
< Content-Type: application/json;charset=ISO-8859-1
< Content-Length: 283
< Server: Jetty(9.2.13.v20150730)
< 
* Connection #0 to host localhost left intact
<massTruncateRequests href="http://localhost:8080/geoserver/gwc/rest/masstruncate.json"> <requestType>truncateLayer</requestType> <requestType>truncateParameters</requestType> <requestType>truncateOrphans</requestType> <requestType>truncateExtent</requestType></massTruncateRequests>```